.gitignore

```diff
@@ -1,6 +1,6 @@
 # Generated by Cargo
 # will have compiled files and executables
-/target/
+target/
```

Comment:
do you know whether it is equivalent to `**/target/`?

Reply:
Not sure what `**/target/` does, but this excludes all directories named `target`.